src/extensions/default/InlineColorEditor/main.js

```diff
@@ -27,7 +27,9 @@ define(function (require, exports, module) {
     var EditorManager       = brackets.getModule("editor/EditorManager"),
         ExtensionUtils      = brackets.getModule("utils/ExtensionUtils"),
         InlineColorEditor   = require("InlineColorEditor").InlineColorEditor,
-        ColorUtils          = brackets.getModule("utils/ColorUtils");
+        ColorUtils          = brackets.getModule("utils/ColorUtils"),
+	 CSSProperties       = require("text!CSSProperties.json"),
+        properties          = JSON.parse(CSSProperties);
```

Comment:
I didn't understand the part "on the first instance where you need it". Could you please elaborate?